workflow/artifacts/oss/oss.go

```diff
@@ -84,6 +84,11 @@ func (p *ossCredentialsProvider) GetCredentials() oss.Credentials {
 
 func (ossDriver *ArtifactDriver) newOSSClient() (*oss.Client, error) {
 	var options []oss.ClientOption
+
+	if proxy, ok := os.LookupEnv("https_proxy"); ok {
```

Comment:
Can you add a comment that this is needed for the OSS driver and that the S3 and GCS drivers respect this environment variable?

Reply:
I have tested other drivers and added doc for oss driver.
please refer #12313 and [oss cli doc](https://help.aliyun.com/zh/cli/use-an-http-proxy-server#section-5yf-ejl-jwf)
thanks